tests/web_server/test_core_endpoints.py

```diff
@@ -140,3 +141,53 @@ async def test_command_overwrite_failure(aiohttp_client, authorized_cookies):
     assert resp.status == HTTPStatus.OK
     config_dict = await resp.json()
     assert config_dict.get('requirements', dict()).get('go', dict()).get('command') == 'go version'
+
+
+async def test_custom_rejecting_login_handler(aiohttp_client):
+    class RejectAllLoginHandler(LoginHandlerInterface):
+        def __init__(self, services):
+            self.auth_svc = services.get('auth_svc')
+            self._name = 'Reject All Logins'
```

Comment:
What's the purpose of the name property? If we're going to use it and the property definition exists on the superclass, we should likely pass it in to the `super()` initializer. Or, make it a class-level variable. Either way, it feels a bit awkward to set a "private" variable that is only used by the superclass.

Reply:
I'll have it passed to the super() initializer, and for the tests I won't bother with the names since they aren't necessary

Reply:
I kept the names, but it'll be done via the constructor